src/pilot/assign_role.py

```diff
@@ -267,15 +267,24 @@ def define_target_raid_config(role, drac_client):
         'logical_disks': logical_disks} if logical_disks is not None else None
 
 
-def is_raid_controller(fqdd):
-    return fqdd.startswith('RAID.')
+def is_raid_controller(fqdd, drac_client):
+    if fqdd.startswith('RAID.'):
+        return True
+    elif fqdd.startswith('BOSS'):
```

Comment:
This method should just return is_boss_controller(fqdd), and the is_boss_controller() method should contain the logic that detects the BOSS controller.

Reply:
okay